MiniApp/Classes/MiniApp.swift

```diff
@@ -32,10 +32,11 @@ public class MiniApp: NSObject {
     ///     -   completionHandler: A block to be called when MiniAppInfo information is fetched. Completion blocks receives the following parameters
     ///         -   MiniAppInfo: MiniAppInfo information.
     ///         -   Error: Error details if fetching is failed.
-    public func info(miniAppId: String, completionHandler: @escaping (Result<MiniAppInfo, Error>) -> Void) {
+    public func info(miniAppId: String, completionHandler: @escaping (Result<MiniAppInfo, MASDKError>) -> Void) {
         if miniAppId.count == 0 {
-            return completionHandler(.failure(NSError.invalidAppId()))
+            return completionHandler(.failure(MASDKError.invalidAppId))
```

Comment:
You can return just `.invalidAppId`. We don't need to use `MASDKError.invalidAppId`

Reply:
Oh, cool. Thanks, I didn't know about that.